prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -242,7 +252,22 @@ TLS_KEY=$(echo "${CERT_KEY}" | tail -1)
 
 shout "Apply Kyma config"
 date
-"${KYMA_SCRIPTS_DIR}"/concat-yamls.sh "${INSTALLER_YAML}" "${INSTALLER_CONFIG}" "${INSTALLER_CR}" \
+
+if [[ "$BUILD_TYPE" == "release" ]]; then
+    echo "Use released artifacts"
+    gsutil cp "${KYMA_ARTIFACTS_BUCKET}/${RELEASE_VERSION}/kyma-config-cluster.yaml" downloaded-kyma-config-cluster.yaml
```

Comment:
Consider a temporary directory for such downloaded files (/tmp/<somedir>/). Chance of collision is low, but it's better not to mix things.